react/features/welcome/components/RootNavigationContainer.js

```diff
@@ -53,6 +55,13 @@ const RootNavigationContainer = ({ isWelcomePageAvailable }: Props) => (
                     component = { DialInSummary }
                     name = { screen.dialInSummary }
                     options = { dialInSummaryScreenOptions } />
+                <RootStack.Screen
+                    component = { ConferenceNavigationContainer }
+                    name = { screen.conference.main }
```

Comment:
I think this is a problem in v6, you cannot have 2 screens with the same name. Well, you can, but it's discouraged.

Reply:
You can introduce a new name like conference.root and have main be the first screen in that navigator by default. See how I changed it when migrating from v5 to v6.

Reply:
Resolved.